attorch/train.py

```diff
@@ -56,7 +61,13 @@ def _objective(mod):
     while patience_counter < patience and epoch < max_iter:
         for _ in range(interval):
             epoch += 1
-            yield epoch, current_objective
+            if time_obj_tracker is None:
```

Comment:
What about making `time_object_tracker` an object that has a function `log_objective(time, obj)`? I think this would be cleaner than returning different return arguments.

Reply:
done